sections/futures/PositionCard/PositionCard.tsx

```diff
@@ -379,6 +352,7 @@ const StyledSubtitle = styled.p`
 	color: ${(props) => props.theme.colors.common.secondaryGray};
 	text-transform: capitalize;
 	margin: 0;
+	cursor: help;
```

Comment:
As @platschi mentioned, there is no need to include the price with the tooltip cursor.